compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -107,9 +91,10 @@ void copy_data<loco::DataType::STRING>(const VectorWrapper<uint8_t> &raw_data,
 namespace luci
 {
 
-CircleConst *create_circleconst(GraphBuilderContext *context, int32_t tensor_index)
```

Comment:
Ahh, you are right, but I made context as second arg because `GraphBuilder::build` makes context as second arg:

```cpp
CircleNode *build(const circle::OperatorT &op, GraphBuilderContext *context) const final;
```